app/CssStats/CssStatsServiceProvider.php

```diff
@@ -22,6 +22,7 @@ public function register(Container $app)
                 [
                     'timeout' => $app['css_stats.timeout'] ?? 5,
                     'connect_timeout' => $app['css_stats.connect_timeout'] ?? 1,
+                    'ssl.certificate_authority' => get_cfg_var('openssl.cafile'),
```

Comment:
I checked the value of `openssl.cafile` on the production server of project aanvraag by doing a `php -i` and then searching for `openssl.cafile`. The value on the production server is
```
Directive => Local Value => Master Value
openssl.cafile => no value => no value
openssl.capath => no value => no value
```
So not sure this will fix the issue without other changes?

Reply:
My Vagrant box has some values for this config
```
Directive => Local Value => Master Value
openssl.cafile => /usr/local/etc/openssl@1.1/cert.pem => /usr/local/etc/openssl@1.1/cert.pem
openssl.capath => /usr/local/etc/openssl@1.1/certs => /usr/local/etc/openssl@1.1/certs
```

Reply:
hah. good catch. I have only tested on the testserver. Will doublecheck the values in prod first, and will comment here.

Reply:
config values set in test and prod. please verify.

Reply:
Looks good now on the servers.